karate-tests/src/test/java/mocks/sandbox/operations.js

```diff
@@ -0,0 +1,60 @@
+function updateAddressDetails (value, originalPatient, updateErrors) {
+  if (value === '2 Whitehall Quay') {
+    updateErrors.push('Invalid update with error - no id or url found for path with root /address/0')
+  } else if (!Object.prototype.hasOwnProperty.call(originalPatient, 'address')) {
```

Comment:
Would this be cleaner?
```suggestion
  } else if (!originalPatient.hasOwnProperty('address')) {
```

Reply:
It sounds like this may fall in to the "When not to use it" in the [doc](https://eslint.org/docs/latest/rules/no-prototype-builtins). But happy to follow the linter. Leave it as is.